src/robot_properties_nyu_finger/config.py

```diff
@@ -130,6 +128,132 @@ class NYUFingerConfig(NYUFingerAbstract):
     v0 = zero(robot_model.nv)
     a0 = zero(robot_model.nv)
 
+class NYUFingerDoubleConfig0(NYUFingerConfig):
+    robot_family = "nyu_finger"
+    robot_name = "nyu_finger_double0"
+
+    paths = find_paths(robot_name)
+    meshes_path = paths["resources"]
+    dgm_yaml_dir = paths["dgm_yaml_dir"]
+    dgm_yaml_path = paths["dgm_yaml"]
+    urdf_path = paths["urdf"]
+
+    # The inertia of a single blmc_motor.
+    motor_inertia = 0.0000045
+
+    # The motor gear ratio.
+    motor_gear_ration = 9.0
```

Comment:
Isn't the `motor_gear_ratio` etc already defined on the `NYUFingerConfig`? So no need to redefine them here?